app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb

```diff
@@ -0,0 +1,21 @@
+<%#
+name: Remote Execution Host Job
+description: Example payload for actions.remote_execution.run_host_job_succeeded
+snippet: false
+model: WebhookTemplate
+-%>
+#
+# Example webhook template, turn this into a XML, JSON, shell script etc.
```

Comment:
In safe mode there is only JSON available through `payload()` macro. Should we re-write the template to more explicit ready to use version/example?

Reply:
I dropped mentioning formats completely.